### ISSUE TITLE
toolbox: failed trying to create container, name must match regex

### ISSUE THREAD

**Issue by zerotri** [debarshiray/toolbox#106]
I'm running a fresh Silverblue 29 install, rebased to 30 via these instructions: https://fedoramagazine.org/how-to-rebase-to-fedora-30-beta-on-silverblue/

A simple toolbox create fails:
```
[zerotri@crescens ~]$ toolbox -v create
toolbox: Fedora generational core is f30
toolbox: base image is fedora-toolbox:30
toolbox: customized user-specific image is fedora-toolbox-zerotri:30
toolbox: container is fedora-toolbox-zerotri:30
toolbox: checking value /var/run/.heim_org.h5l.kcm-socket (Stream) of property Listen in sssd-kcm.socket
toolbox: parsing value /var/run/.heim_org.h5l.kcm-socket (Stream) of property Listen in sssd-kcm.socket
toolbox: checking if image fedora-toolbox-zerotri:30 already exists
Error: error getting image "fedora-toolbox-zerotri:30": unable to find a name and tag match for fedora-toolbox-zerotri in repotags: no such image
toolbox: looking for image localhost/fedora-toolbox:30
Pulling docker://localhost/fedora-toolbox:30
ERRO[0000] exit status 1                                
toolbox: looking for image registry.fedoraproject.org/f30/fedora-toolbox:30
Pulling docker://registry.fedoraproject.org/f30/fedora-toolbox:30
Getting image source signatures
Copying blob 3a46889e9453: 43.45 MiB / 89.09 MiB [==============>--------------]
Copying blob 3a46889e9453: 82.37 MiB / 89.09 MiB [==========================>--]
Copying blob 3a46889e9453: 89.09 MiB / 89.09 MiB [==========================] 5s
Copying blob 419d41ab508b: 59.05 MiB / 59.05 MiB [==========================] 5s
Copying config b8e18bacde99: 2.68 KiB / 2.68 KiB [==========================] 0s
Writing manifest to image destination
Storing signatures
toolbox: base image fedora-toolbox:30 resolved to registry.fedoraproject.org/f30/fedora-toolbox:30
toolbox: trying to create working container toolbox-working-container-ccae51ea-59db-11e9-a458-646e69d4a619
toolbox: trying to configure working container toolbox-working-container-ccae51ea-59db-11e9-a458-646e69d4a619
passwd: Note: deleting a password also unlocks the password.
passwd: Note: deleting a password also unlocks the password.
toolbox: trying to create image fedora-toolbox-zerotri:30
Getting image source signatures
Skipping blob 9d8f0e640139 (already present): 296.86 MiB / 296.86 MiB [=====] 0s
Skipping blob 181c802fea2f (already present): 170.15 MiB / 170.15 MiB [=====] 0s
Copying blob 8d99a44be6b3: 306.50 KiB / 306.50 KiB [========================] 0s
Skipping blob 9d8f0e640139 (already present): 296.86 MiB / 296.86 MiB [=====] 0s
Skipping blob 181c802fea2f (already present): 170.15 MiB / 170.15 MiB [=====] 0s
Copying blob 8d99a44be6b3: 306.50 KiB / 306.50 KiB [========================] 0s
Copying config b73fa169533d: 1.38 KiB / 1.38 KiB [==========================] 0s
Writing manifest to image destination
Storing signatures
toolbox: created image fedora-toolbox-zerotri:30
toolbox: checking if image fedora-toolbox-zerotri:30 has volumes for host bind mounts
toolbox: checking if container fedora-toolbox-zerotri:30 already exists
Error: error looking up container "fedora-toolbox-zerotri:30": no container with name or ID fedora-toolbox-zerotri:30 found: no such container
toolbox: trying to create container fedora-toolbox-zerotri:30
Error: error running container create option: name must match regex [a-zA-Z0-9_-]+: invalid argument
toolbox: failed to create container fedora-toolbox-zerotri:30
```

Podman is version 1.2.0

I also checked `~/.config/containers/storage.conf` and am not encountering any issues with CamelCase settings described in other issues.

The regex being invalid seems unusual as the container naming scheme seems to match that of others I can find listed in issues.

EDIT: After a bit of playing around, passing the name `fedora-toolbox-zerotri` with the `--container` argument succeeds in creating and starting the container. It seems to be the colon in the name that podman doesn't like.

**Comment by debarshiray** [comment]
Yes, this is due to a very recent `podman` change and has been discussed in https://github.com/containers/libpod/issues/2876

As you found out, we need to update the toolbox to comply with the new rules. PRs welcome.

**Comment by debarshiray** [comment]
Should be fixed by #113. Feel free to leave a comment or re-open this issue if something is wrong.
